Patch 1.22.1 (#950)
LayerManager/layman: CHANGELOG.md

# Changelog

## v1.22.1
 2023-10-12
### Changes

> - [#949](https://github.com/LayerManager/layman/issues/949) Fix map-layer relation migration for map compositions including other than `WMS` and `Vector` layers. 

Reviewer: ```suggestion
- [#949](https://github.com/LayerManager/layman/issues/949) Fix reading map-layer relations from map composition that includes also other than WMS and Vector layers.
```
